charm.go

```diff
@@ -38,3 +40,29 @@ func ReadCharm(path string) (charm Charm, err error) {
 	}
 	return charm, nil
 }
+
+// SeriesToUse takes a specified series and a list of series supported by a
+// charm and returns the series which is relevant.
+// If the specified series is empty, the any default series defined by the
```

Comment:
s/the any/then any/

Is it really *any*? you're selecting the first one, and in the code you say "charm default". Is the charm default the first value in the list? If so, you should not say "any" here.